docs/embeddings.md

```diff
@@ -135,22 +171,27 @@ openai_ef = embedding_functions.OpenAIEmbeddingFunction(
 <TabItem value="js" label="JavaScript">
 
 ```javascript
-const {OpenAIEmbeddingFunction} = require('chromadb');
-const embedder = new OpenAIEmbeddingFunction({openai_api_key: "apiKey"})
+const { OpenAIEmbeddingFunction } = require('chromadb');
+const embedder = new OpenAIEmbeddingFunction({ openai_api_key: 'apiKey' });
 
-// use directly 
-const embeddings = embedder.generate(["document1","document2"])
+// use directly
+const embeddings = embedder.generate(['document1', 'document2']);
```

Comment:
I think `embeddings` can be removed as it is not used in the example.

Reply:
Alternatively, add an example of of how embeddings can be inserted into the collection being created.